theme-data/common/common.json

```diff
@@ -4,6 +4,7 @@
       "text": {
         "white": "@color-white-alpha-95",
         "black": "@color-black-alpha-95",
+        "gray": "@color-gray-70",
```

Comment:
I see "white-alpha-70" on the figma assigned to commont-text-gray

Reply:
**Nice catch!**
It was a mistake, I have interpreted rgba(255,255,255,0.7) as being gray, but it is white.
Pushed commit
[Changed common-text-gray from @color-gray-70 to @color-white-alpha-70](https://github.com/momentum-design/tokens/pull/174/commits/62833c412193fdf3886b25d6b51f5531c99cb438)
Changing this conversation to Resolved.